adhocracy4/ckeditor/fields.py

```diff
@@ -1,23 +1,10 @@
 from ckeditor.fields import RichTextField
 from ckeditor_uploader.fields import RichTextUploadingField
 
-_extra_plugins = ["collapsibleItem", "embed", "embedbase"]
-_external_plugin_resources = [
-    (
-        "collapsibleItem",
-        "/static/ckeditor_collapsible/",
-        "plugin.js",
-    )
-]
-
 
+# FIXME: remove these fields / file
 class RichTextCollapsibleMixin:
-    def __init__(self, *args, **kwargs):
-        kwargs["extra_plugins"] = kwargs.get("extra_plugins", []) + _extra_plugins
-        kwargs["external_plugin_resources"] = (
-            kwargs.get("external_plugin_resources", []) + _external_plugin_resources
-        )
-        super().__init__(*args, **kwargs)
+    pass
 
 
```

Comment:
not sure how to get rid of these fields, as they will break old migrations. One way would be to squash / reset migrations but that also has its drawbacks

Reply:
@m4ra any idea here?

Reply:
The fields that will be deleted, the data they retain they don't need to be migrated to the new CK?

Reply:
They are migrated to `CKEditor5Field`. Will look into squashing, thanks!